Gemfile

```diff
@@ -20,7 +20,7 @@ gem "cheffish", ">= 17"
 group(:omnibus_package) do
   gem "appbundler"
   gem "rb-readline"
-  gem "inspec-core-bin", "~> 4.24" # need to provide the binaries for inspec
+  gem "inspec-core-bin", "~> 5" # need to provide the binaries for inspec
```

Comment:
A bit concerned with `~> 5` as a spec here. Can we add a minor version?

Reply:
updated